images/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.21-openshift-4.16 as builder
```

Comment:
Can we leave a comment in this file to explain what it is used for to ensure the context of what this dockerfile is known

Reply:
How about moving the filename to `Dockerfile.e2e` and leaving `Dockerfile` for regular usage?